src/domain/conversation.js

```diff
@@ -64,6 +64,26 @@ export default class Conversation {
       },
       include: { messages: true, users: true }
     })
+    foundConversations = await Promise.all(
+      foundConversations.map(async (conversation) => {
+        let newMessages = await dbClient.message.findMany({
+          where: {
+            conversationId: conversation.id
+          },
+          include: {
+            user: { include: { profile: true } }
+          }
+        })
+        newMessages = newMessages.map((newMessage) => {
+          const createdBy =
+            newMessage.user.profile.firstName +
+            ' ' +
```

Comment:
Can we change this to string interpolation please?

Reply:
Just changed it